repository/src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.esm.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.*;
+
+import javax.persistence.*;
```

Comment:
Try to avoid wildcards in import statements.

Reply:
I try to avoid  wildcards in import statements but Intellige Idea use it without my decision when the one package import number  is more then 5 or 6

Reply:
You can setup this number in settings "File | Settings | Editor | Code Style | Java | Class count to use import with '*'".